testsuite/features/secondary/sle_minion_bootstrap_ssh_key.feature

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2016-2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to bootstrap a Salt minion via the GUI using SSH key
+
+  Scenario: Delete SLES minion system profile before bootstrap test
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    And I cleanup minion "sle_minion"
+    Then "sle_minion" should not be registered
+
+  Scenario: Prepare the minion for SSH key authentication
+    When I backup the SSH authorized_keys file of host "sle_minion"
+    And I add pre-generated SSH public key to authorized_keys of host "sle_minion"
+  
+  Scenario: Bootstrap a SLES minion using SSH Key with wrong passphrase
+    Given I am authorized
+    When I go to the bootstrapping page
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "sle_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I check radio button "SSH Private Key"
+    And I attach the file "ssh_keypair/id_rsa_bootstrap-passphrase_linux" to "privKeyFile"
+    And I enter "you-shall-not-pass" as "privKeyPwd"
+    And I click on "Bootstrap"
+    And I wait until I see "Permission denied, no authentication information" text
+
+  Scenario: Bootstrap a SLES minion using SSH Key
+    Given I am authorized
+    When I go to the bootstrapping page
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "sle_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I check radio button "SSH Private Key"
+    And I attach the file "ssh_keypair/id_rsa_bootstrap-passphrase_linux" to "privKeyFile"
+    And I enter "linux" as "privKeyPwd"
+    And I select the hostname of "proxy" from "proxies"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+
+  Scenario: Check new minion bootstrapped with SSH Key in System Overview page
+    Given I am authorized
+    When I go to the minion onboarding page
+    Then I should see a "accepted" text
+    When I navigate to "rhn/systems/Overview.do" page
```

Comment:
I personally prefer to don't have calls to endpoints in out steps, I don't remember now the step, but we have another that has the same behavior without having the URI there.

Reply:
100 % agree, we should avoid internals in high-level cucumber text.

Reply:
Looking at the test codebase, it even seems this seems to be the way to navigate to a "general" page. You are right, that there is another way - it's used in docker building feature (and it's tighly related to it), but the problem is, we'd need to write extra step for each navigation target. Btw, my code was a c&p from an existing bootstrapping test. And while I also don't like having `rhn/blabla/bar` in the steps that much, this seems like this should be done separately as a refactoring that targets all occurences of this step: `When I navigate to "rhn/systems/Overview.do" page`.

```bash
git grep 'I navigate to' | grep  "\.do" | wc -l
> 22

git grep 'I navigate to' | grep -v "\.do" | wc -l
> 11
```